### DISCUSSION TITLE
Git pull error

### DISCUSSION THREAD

**Discussion by KarlaAns** [community/community#40294] [Other Feature Feedback, Questions, & Ideas]
Question

### Body

So  I did a git pull in a normal folder that wasn't even a repository (after that a hint appeared and I put that in the Command Prompt)  but now my file browser is broken, it says that is loading but nothing happens and idk how to fix that, I can't enter with the file browser, what can I do?
This is what it shows:
![problemWithFileBrowser](https://user-images.githubusercontent.com/116769089/204350397-dbad4e70-c81a-47cc-bfbb-8569a1791cff.jpg)

**Comment by ossanna16** [comment]
Hi there @KarlaAns and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
